src/app/settings/views/Configuration/CommissioningFormFields/CommissioningFormFields.tsx

```diff
@@ -71,6 +71,7 @@ const CommissioningFormFields = (): JSX.Element => {
         type="text"
       />
       <FormikField
+        aria-label="K_g BMC key textbox"
```

Comment:
I think if you use my suggestion for the regex to do a partial match on the label then this aria-label won't be necessary.

Reply:
https://github.com/canonical-web-and-design/maas-ui/pull/4265#discussion_r926521825 <-- Going to link to my comment on that suggestion, TL;DR: I'm not sure it's possible because the label and field aren't rendered together in the DOM